pkg/planner/core/physical_plans.go

```diff
@@ -1119,11 +1119,6 @@ func expandVirtualColumn(schema *expression.Schema, copyColumn []*model.ColumnIn
 	return copyColumn
 }
 
-// SetIsChildOfIndexLookUp is to set the bool if is a child of IndexLookUpReader
-func (ts *PhysicalTableScan) SetIsChildOfIndexLookUp(isIsChildOfIndexLookUp bool) {
-	ts.isChildOfIndexLookUp = isIsChildOfIndexLookUp
```

Comment:
this variable is for two kind of usage, one is for explainID usage, one is for getCost adjustment.